README.md

```diff
@@ -36,4 +39,4 @@ OWT project is licensed under Apache License, Version 2.0. By contributing to th
 Use the "Issues" tab on Github.
 
 ## See Also
-http://webrtc.intel.com
+https://webrtc.intel.com
```

Comment:
link with https:// is not available.

Reply:
Oops, it doesn't work. Restored to http. But it redirects to a https page.